internal/operators/ocs/validations.go

```diff
@@ -116,10 +85,6 @@ func (o *ocsValidator) ValidateRequirements(cluster *models.Cluster) (api.Valida
 			if host.Role == models.HostRoleAutoAssign {
 				status = "All host roles must be assigned to enable OCS."
 				o.log.Info("Setting Operator Status ", status)
```

Comment:
please change the log to: "Validate Requirements status:"

Reply:
fixed